### ISSUE TITLE
Wikipedia connector disambiguation can lead to unexpected results

### ISSUE THREAD

**Issue by cragwolfe** [Unstructured-IO/unstructured#330]
Running:

```
$ PYTHONPATH=. ./unstructured/ingest/main.py \
    --wikipedia-page-title "Mango" \
    --structured-output-dir wikipedia-ingest-output \
    --num-processes 2 --download-dir wikipedia-download --preserve-downloads --reprocess \
    --verbose
```

leads to results for a `Manga` page, not the requested `Mango` page, probably due the default `auto_suggest=True` keyword argument in `wikipedia.page`. Reference: https://wikipedia.readthedocs.io/en/latest/code.html#module-wikipedia .

There should be a way to fetch the intended page. One may be to add a `wikipedia-auto-suggest` param to 
https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/ingest/main.py

**Comment by tomaarsen** [comment]
I encountered the `auto_suggest` parameter while implementing the data connector, but I expected that leaving it as `True` would be sufficient. However, that has shown to be wrong.

I can either add the CLI argument as you suggested tomorrow, or I can have a look at an alternative Python module for accessing the Wikipedia API. I found out that the one that we rely on now has not been updated in a few years.
That said, I'm not sure whether the redirection oddity is a consequence of a bug/flaw in the Python module, or whether it originates on the Wikipedia API side. If it's the latter, than switching Python modules won't actually do anything obviously.

- Tom Aarsen
